modules/op-release-notes-1-14.adoc

```diff
@@ -0,0 +1,269 @@
+= Release notes for {pipelines-title} General Availability 1.14
+
+With this update, {pipelines-title} General Availability (GA) 1.14 is available on {OCP} 4.12 and later versions.
+
+[id="new-features-1-14_{context}"]
+== New features
+
+In addition to fixes and stability improvements, the following sections highlight what is new in {pipelines-title} 1.14:
+
+[NOTE]
+====
+Before upgrading to the {pipelines-title} Operator 1.14, ensure that you installed at least the {OCP} 4.12.19 or 4.13.1 version on your cluster.
+====
+
+[id="pipelines-new-features-1-14_{context}"]
+=== Pipelines
+
+* With this update, you can use a parameter of a task or pipeline or a result of a previous task to specify the name of a resource to bind to a workspace, for example, `name: $(params.name)-configmap`.
+
+* With this update, {pipelines-shortname} supports using your existing entitlements for Red Hat Enterprise Linux in build processes within your pipelines. The built-in `buildah` cluster task can now use these entitlements.
+
+* With this update, if a pipeline run or task run uses the `pipeline` service account, you can use CSI volume types in the pipeline or task.
+
+* With this update, you can use a `StepAction` custom resource (CR) to define a reusable scripted action that you can invoke from any number of tasks. To use this feature, you must set the `pipeline.options.configMaps.feature-flags.data.enable-step-actions` spec to `true`.
+
+* With this update, object parameters and array results are enabled by default. You do not need to set any flags to use them.
+
+* With this update, you can use the HTTP resolver to fetch a pipeline or task from a HTTP URL.
++
+.Example usage for a task
+[source,yaml]
+----
+apiVersion: tekton.dev/v1
+kind: TaskRun
+metadata:
+  name: remote-task-reference
+spec:
+  taskRef:
+    resolver: http
+    params:
+    - name: url
+      value: https://raw.githubusercontent.com/tektoncd-catalog/git-clone/main/task/git-clone/git-clone.yaml
+----
++
+.Example usage for a pipeline
+[source,yaml]
+----
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  name: http-demo
+spec:
+  pipelineRef:
+    resolver: http
+    params:
+    - name: url
+      value: https://raw.githubusercontent.com/tektoncd/catalog/main/pipeline/build-push-gke-deploy/0.1/build-push-gke-deploy.yaml
+----
+
+* With this update, you can use an enum declaration to limit the values that you can supply for a parameter of a pipeline or task. To use this feature, you must set the `pipeline.options.configMaps.feature-flags.data.enable-param-enum` spec to `true`.
++
+.Example usage
+[source,yaml]
+----
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: pipeline-param-enum
+spec:
+  params:
+  - name: message
+    enum: ["v1", "v2"]
+    default: "v1"
+# ...
+----
+
+* With this update, when using the Git resolver with the authenticated SCM API, you can override the default token, SCM type, and server URL that you configured.
```

Comment:
```suggestion
* With this update, when using the Git resolver with the authenticated SCM API, you can override the default token, SCM type, and server URL that you configured. See the following example:
```

Reply:
I can't find the specific guidance, but all of the examples I've seen in the OpenShift guidelines have an introduction before examples (e.g., "in the following example"), so I generally try to follow that.